stylelint.config.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
   extends: './node_modules/wttj-config/lib/stylelint',
+  customSyntax: 'postcss-styled-syntax',
```

Comment:
What is it for? Can't find it in the panda doc

Reply:
not related to panda, cf https://github.com/WTTJ/front-config/pull/214